crates/consensus/src/receipt/receipts.rs

```diff
@@ -28,30 +28,12 @@ impl Receipt {
     }
 }
 
-impl alloy_network::Receipt for Receipt {
```

Comment:
These impls have been moved to the network crate because of a circular dep.

Note to self: we can move this trait to `alloy-consensus` too, so the impl is colocated